src/System.Net.Http.WinHttpHandler/tests/project.lock.json

```diff
@@ -173,10 +173,10 @@
           "lib/dotnet/System.IO.FileSystem.Primitives.dll": {}
         }
       },
-      "System.Linq/4.0.0-beta-22816": {
+      "System.Linq/4.0.0-beta-22703": {
         "dependencies": {
-          "System.Collections": "4.0.10-beta-22816",
-          "System.Runtime": "4.0.20-beta-22816"
+          "System.Collections": "4.0.0-beta-22703",
+          "System.Runtime": "4.0.0-beta-22703"
```

Comment:
Yeah, that is https://github.com/dotnet/corefx/issues/1848.  Still odd that it would rollback, that raises the question of where the previous version was even coming from.  If you'd like to avoid this you could add a direct reference to that package with a * version.